task_processing/plugins/mesos/execution_framework.py

```diff
@@ -223,6 +224,11 @@ def offer_matches_pool(self, offer):
 
         return False, None
 
+    def offer_matches_task_constraints(self, offer, task):
```

Comment:
you could move this to ``constraints.py`` - it doesn't need to be part of this object